### ISSUE TITLE
Allow configuring null values similarly to @CsvSource

### ISSUE THREAD

**Issue by mikemybytes** [mikemybytes/junit5-formatted-source#3]
JUnit 5 `@CsvSource` allows configuring null values using a dedicated annotation parameter:
```java
public @interface CsvSource {
    // ...

    	/**
	 * A list of strings that should be interpreted as {@code null} references.
	 *
	 * <p>For example, you may wish for certain values such as {@code "N/A"} or
	 * {@code "NIL"} to be converted to {@code null} references.
	 *
	 * <p>Please note that <em>unquoted</em> empty values will always be converted
	 * to {@code null} references regardless of the value of this {@code nullValues}
	 * attribute; whereas, a <em>quoted</em> empty string will be treated as an
	 * {@link #emptyValue}.
	 *
	 * <p>Defaults to {@code {}}.
	 *
	 * @since 5.6
	 */
	String[] nullValues() default {};

    // ...
}
```

A similar approach could be adopted here.

**Comment by mikemybytes** [comment]
To be released with the next library version.

**Comment by mikemybytes** [comment]
Released in [0.2.0](https://github.com/mikemybytes/junit5-formatted-source/releases/tag/0.2.0).
